src/layout/Navigation.tsx

```diff
@@ -409,6 +409,18 @@ export const Navigation: FunctionComponent<PropsWithChildren> = ({
         window.open("https://v2018.mesbg-list-builder.com/", "_blank").focus(),
       active: false,
     },
+    {
+      icon: <OpenInNew />,
```

Comment:
Im kinda unsure about the icon. This could be fine, but maybe another? 

@avcordaro what is your opinion?

Reply:
Changed the icon to an Announcement icon.